src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -126,7 +126,10 @@ public function getRouteCollectionForRequest(Request $request)
                     $portal->getWebspace()->getKey(),
                     $language
                 );
-                if (
+
+                if (!$content) {
+                    throw new ResourceLocatorNotFoundException();
+                } elseif (
```

Comment:
Actually this change was necessary, because there was no document loaded at all if the content in this language is gone. After viewing this file I recognized that it could be significantly simplified due to the new publishing feature (because we are already loading only published documents anyway). Should I do that now, or create an issue for that? /cc @chirimoya